packages/frontend/src/features/sqlRunner/components/visualizations/Table.tsx

```diff
@@ -12,26 +12,21 @@ import {
     Tr,
 } from '../../../../components/common/Table/Table.styles';
 import { type useSqlQueryRun } from '../../hooks/useSqlQueryRun';
-import { useAppSelector } from '../../store/hooks';
 import { useTableDataTransformer } from '../../transformers/useTableDataTransformer';
 
 type Props = {
     data: NonNullable<ReturnType<typeof useSqlQueryRun>['data']>;
     config?: TableChartSqlConfig;
 };
 
-export const Table: FC<Props> = ({ data }) => {
-    const resultsTableConfig = useAppSelector(
-        (state) => state.sqlRunner.tableChartConfig,
-    );
-
+export const Table: FC<Props> = ({ data, config }) => {
```

Comment:
so we can reuse this viz on any component since it doesn't rely on RTK selectors